docs/reference/query-dsl/rank-feature-query.asciidoc

```diff
@@ -120,31 +159,83 @@ GET test/_search
+----
 // CONSOLE
 
-[float]
-=== Supported functions
 
-The `rank_feature` query supports 3 functions in order to boost scores using the
-values of rank features. If you do not know where to start, we recommend that you
-start with the `saturation` function, which is the default when no function is
-provided.
+[[rank-feature-top-level-params]]
+==== Top-level parameters for `rank_feature`
+
+`field`::
+(Required, string) <<rank-feature,`rank_feature`>> or
+<<rank-features,`rank_features`>> field used to boost
+<<query-filter-context,relevance scores>>.
+
+`boost`::
++
+--
+(Optional, float) Floating point number used to decrease or increase
+<<query-filter-context,relevance scores>>. Defaults to `1.0`.
+
+Boost values are relative to the default value of `1.0`. A boost value between
+`0` and `1.0` decreases the relevance score. A value greater than `1.0`
+increases the relevance score.
+--
+
+`saturation`::
++
+--
+(Optional, <<rank-feature-query-saturation,function object>>) Saturation
+function used to boost <<query-filter-context,relevance scores>> based on the
+value of the rank feature `field`. If no function is provided, the `rank_feature`
+query defaults to the `saturation` function. See
+<<rank-feature-query-saturation,Saturation>> for more information.
+
+You cannot use the `saturation` parameter with the `log` or `sigmoid`
+parameters.
+--
+
+`log`::
++
+--
+(Optional, <<rank-feature-query-logarithm,function object>>) Logarithmic
+function used to boost <<query-filter-context,relevance scores>> based on the
+value of the rank feature `field`. See
+<<rank-feature-query-logarithm,Logarithm>> for more information.
 
-[float]
-==== Saturation
+You cannot use the `log` parameter with the `saturation` or `sigmoid`
+parameters.
+--
 
-This function gives a score that is equal to `S / (S + pivot)` where `S` is the
-value of the rank feature and `pivot` is a configurable pivot value so that the
-result will be less than +0.5+ if `S` is less than pivot and greater than +0.5+
-otherwise. Scores are always is +(0, 1)+.
+`sigmoid`::
++
+--
+(Optional, <<rank-feature-query-sigmoid,function object>>) Sigmoid function used
+to boost <<query-filter-context,relevance scores>> based on the value of the
+rank feature `field`. See <<rank-feature-query-sigmoid,Sigmoid>> for more
+information.
 
-If the rank feature has a negative score impact then the function will be computed as
-`pivot / (S + pivot)`, which decreases when `S` increases.
+You cannot use the `sigmoid` parameter with the `saturation` or `log`
+parameters.
+--
+
+
+[[rank-feature-query-notes]]
+==== Notes
+
+[[rank-feature-query-saturation]]
+===== Saturation
+The `saturation` function gives a score equal to `S / (S + pivot)`, where `S` is
+the value of the rank feature field and `pivot` is a configurable pivot value so
+that the result will be less than `0.5` if `S` is less than pivot and greater
+than `0.5` otherwise. Scores are always between `0` and `1`.
```

Comment:
Perhaps we could keep the notation `(0, 1)` or say "Scores are always between `0` and `1` (exclusive)"?

Reply:
Re-added with 890b6b3